modules/cnf-about-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_irq_affinity_setting_{context}"]
+= About support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask. These IRQ controllers effectively run on CPU 0.
+
+The following list by no means exhaustive are examples of drivers and hardware that Red Hat are aware lack support for IRQ affinity setting:
```

Comment:
Replace with this suggestion:
"The following are examples of drivers and hardware that Red Hat are aware lack support for IRQ affinity setting. The list is, by no means, exhaustive."